assets/js/lib/operations/index.js

```diff
@@ -6,12 +6,15 @@ export const HOST_OPERATION = 'host';
 export const CLUSTER_OPERATION = 'cluster';
 export const SAP_SYSTEM_OPERATION = 'sap_system';
 export const APPLICATION_INSTANCE_OPERATION = 'application_instance';
+export const CLUSTER_HOST_START_OPERATION = 'cluster_host_start';
```

Comment:
We don't need these 2 new scopes.
These initial entries are the operation scopes, and we already have the `CLUSTER_HOST_OPERATION`, which I think includes the new operations